### ISSUE TITLE
Docker container tests with external registery failing

### ISSUE THREAD

**Issue by san7ket** [SatelliteQE/robottelo#5735]
I see that the related BZ was Verified with snap27 , but the related tests are still failing for snap30
@lpramuk @rplevka any pointers on this ?

- [ ] tests.foreman.cli.test_docker.DockerContainersTestCase
- [ ] tests.foreman.ui.test_docker.DockerContainerTestCase
- [ ] tests.foreman.api.test_docker.DockerContainerTestCase

UI/CLI/API test fail with 
```
LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv docker container create --name="JsbrbBbf4X" --organization-ids="24" --repository-name="rhel" --tag="latest" --command="top" --registry-id="5" --compute-resource-id="2"
2017-12-21 22:54:43 - robottelo.ssh - INFO - <<< stderr
[ERROR 2017-12-21 22:54:43 API] 500 Internal Server Error
[ERROR 2017-12-21 22:54:43 Exception] Permission denied - connect(2) for IP:2375 (Errno::EACCES)
Could not create the container:
  Permission denied - connect(2) for 10.8.30.66:2375 (Errno::EACCES)
[ERROR 2017-12-21 22:54:43 Exception] 
```

**Comment by abalakh** [comment]
Worth mentioning - it's not just about external registry, docker containers tests like 
* [ ] tests.foreman.cli.test_docker.DockerContainersTestCase.test_positive_delete_by_id 
* [ ] tests.foreman.cli.test_docker.DockerContainersTestCase.test_positive_create_with_compresource

are failing as well with the same message

**Comment by lpramuk** [comment]
It is caused by selinux denial since there is a bug in ```container-selinux```
https://bugzilla.redhat.com/show_bug.cgi?id=1527052 no assigned port to container_port_t label
it missed to assign any port to container_port_t label which foreman-selinux uses.

Until it is fixed by ```rhel-7.5``` in container-selinux rpm we can/have to use following workaround:
```semanage port -a -t container_port_t -p tcp 2375```

**Comment by lpramuk** [comment]
It was/is real bug (both selinux and upgrade)
- https://bugzilla.redhat.com/show_bug.cgi?id=1563766 upgraded foreman-selinux has no label for 2375/tcp

if still failing decorate tests with this BZ
